.github/workflows/test-all.yml

```diff
@@ -146,3 +178,49 @@ jobs:
             ./test-summary
             ./tests-out/videos
             ./tests-out/screenshots
+
+      - name: Communicate BIT failure on originating issue
+        uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea # v7.0.1
+        if: failure() && github.event.client_payload.origin_issue
```

Comment:
I intentionally did not use step failure (of, for example, the `playwright-tests` step), so that we still report back if BIT failed somewhere else. If we find this creates too much noise, we can target the step failure (by id) instead.

If the action is cancelled, neither case fires.